AndrewRLloyd88/mb-career-accelerator-spaceX
# fixes/data fixes

- expand on typing for launchslides so it is not type annotated as any and now as LaunchObject
- edit date inline
- update method toDateString()
- formatted money using US currency, active/inactive, linked wikipedia article and removed id as not necessary for Rocket component
- make changes to fairing to accomodate recovery ships data
- add link to ship information from ship name
- determine if there are any recovery ships or not and show the ships component conditionally
- upload correct falcon 91.1 image and add some checks based on flight information and flight numbers to image
- correct output for falcon9v1.2
- added conditional check for details as data is not always present
- comment out launchid as information not meaniningful for user
